shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/network/helpers/JsonApiExceptionError.kt

```diff
@@ -0,0 +1,10 @@
+package co.nimblehq.blisskmmic.data.network.helpers
+
+import co.nimblehq.jsonapi.model.JsonApiException
+
+fun Throwable.jsonApiException() : String? {
```

Comment:
the extension name is incorrect I think. Should it be `toErrorMessage` or `buildErrorMessage`?